extensions/Aness6040/Aness_Minecraft_Utils.js

```diff
@@ -0,0 +1,257 @@
+  
+      try {
+        const response = await fetch(`https://api.mcsrvstat.us/2/${ip}:${port}`);
+        const data = await response.json();
+  
+        return data.online;
+      } catch (error) {
+        return false;
+      }
+    }
+  
+    async numberOfPlayers(args) {
+      const ip = args.IP;
+      const port = args.PORT;
+  
+      try {
+        const response = await fetch(`https://api.mcsrvstat.us/2/${ip}:${port}`);
+        const data = await response.json();
+  
+        if (data.players) {
+          return data.players.online;
+        } else {
+          return 0;
+        }
+      } catch (error) {
+        return 'Error';
+      }
+    }
+  
+    async maxPlayers(args) {
+      const ip = args.IP;
+      const port = args.PORT;
+  
+      try {
+        const response = await fetch(`https://api.mcsrvstat.us/2/${ip}:${port}`);
+        const data = await response.json();
+  
+        if (data.players) {
+          return data.players.max;
+        } else {
+          return 0;
+        }
+      } catch (error) {
+        return 'Error';
+      }
+    }
+  
+    async accountExists(args) {
+      const player = args.PLAYER;
+  
+      try {
+        const response = await fetch(`https://api.ashcon.app/mojang/v2/user/${player}`);
+        const data = await response.json();
+  
+        return !data.error;
+      } catch (error) {
+        return false;
+      }
+    }
+  
+    async getServerIconUri(args) {
+      const ip = args.IP;
+      const port = args.PORT;
+  
+      try {
+        const response = await fetch(`https://api.mcsrvstat.us/2/${ip}:${port}`);
+        const data = await response.json();
+  
+        if (data.icon) {
+          return data.icon;
+        } else {
+          return 'No Icon';
+        }
+      } catch (error) {
+        return 'Error';
+      }
+    }
+  
+    async serverMotd(args) {
+      const ip = args.IP;
+      const port = args.PORT;
+  
+      try {
+        const response = await fetch(`https://api.mcsrvstat.us/2/${ip}:${port}`);
+        const data = await response.json();
+  
+        if (data.motd) {
+          if (typeof data.motd === 'string') {
+            return data.motd;
+          } else if (data.motd.clean && Array.isArray(data.motd.clean)) {
+            return data.motd.clean.join('\n');
+          } else if (data.motd.text) {
+            return data.motd.text;
+          }
+        }
+  
+        return 'No MOTD available';
+      } catch (error) {
+        return 'Error';
+      }
+    }
+  
+    getPlayerTextureUrl(args) {
+      const player = args.PLAYER;
+      const textureType = args.TEXTURELIST;
+  
+      let url;
+      if (textureType === 'face') {
+        url = `https://minotar.net/helm/${player}/600.png`;
+      } else if (textureType === 'bust') {
+        url = `https://minotar.net/armor/bust/${player}/500.png`;
+      } else if (textureType === 'cube head') {
+        url = `https://cravatar.eu/helmhead/${player}/600.png`;
```

Comment:
Any reason this option uses a different API than the other 3? Remember, the more APIs the extension relies on, the more likely one may go down.